components/home/spotlight-hero.js

```diff
@@ -1,28 +1,44 @@
-import { Hero } from "@/components/hero";
-import { twJoin } from "tailwind-merge";
+import { Hero, HeroCaption, HeroLink, HeroTitle } from "@uoguelph/react-components/hero";
+import Image from "next/image";
+import { tv } from "tailwind-variants";
 
-export const SpotlightHero = ({ hero }) => (
-  <Hero
-    id={`uofg-homepage-spotlight-hero`}
-    variant="spotlight"
-    title={<h2 className="mt-0">{hero.title}</h2>}
-    image={{
-      src: hero.image.image.url,
-      alt: hero.image.image.alt,
-      width: hero.image.image.width,
-      height: hero.image.image.height,
-      className: twJoin(
-        "aspect-[3/2] w-full spotlight-hero-image",
-        hero.thumbnailImageCrop === "right" && "object-right",
-        hero.thumbnailImageCrop === "left" && "object-left",
-        (hero.thumbnailImageCrop === "center" || !hero.thumbnailImageCrop) && "object-center"
-      ),
-    }}
-    caption={hero.caption}
-    button={{
-      body: hero.url.title,
-      href: hero.url.url,
-    }}
-    alignment={hero.captionAlignment}
-  />
-);
+export const SpotlightHero = ({ data }) => {
+  const classes = tv({
+    slots: {
+      hero: "aspect-3/2 w-full",
+      title: "mt-0! font-sans!",
+    },
+    variants: {
+      thumbnailImageCrop: {
+        left: { hero: "object-left" },
+        right: { hero: "object-right" },
+        center: { hero: "object-center" },
+      },
+    },
+  });
+
+  const { hero, title } = classes({
+    thumbnailImageCrop: data.thumbnailImageCrop ?? "center",
+  });
+
+  return (
+    <Hero
```

Comment:
On narrow viewport, the spotlight hero alignment does not match production:

Production:
<img width="568" alt="image" src="https://github.com/user-attachments/assets/dc7d57cc-cf49-4e9c-8ee6-584f48bb2189" />

Current branch:
<img width="570" alt="image" src="https://github.com/user-attachments/assets/3f742243-381b-4537-b0b4-c96e9525907c" />